routes/foodtrucks.go

```diff
@@ -26,6 +27,20 @@ type addFoodTruckRequest struct {
 	Tags        []string      `json:"tags"`
 }
 
+type updateFoodTruckRequest struct {
+	Name        string       `json:"name" bson:"name"`
+	Address     string       `json:"address" bson:"address"`
+	Location    [2]float64   `json:"location" bson:"location"`
+	Status      bool         `json:"status" bson:"status"`
+	Hours       [7][2]string `json:"hours" bson:"hours"`
+	Reviews     []string     `json:"reviews" bson:"reviews"`
```

Comment:
Review's shouldn't be able to be updated?

Reply:
k ill remove it

Reply:
k ill remove it

Reply:
oh wait i never even had a check whether reviews was in the request body, so this is good. but ill remove it from the struct for clarity